packages/terra-collapse/tests/nightwatch/CustomizedButtonCollapse.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import Collapse from '../../lib/Collapse';
+
+const ButtonAttrsCollapse = () => (
+  <Collapse closedButtonText="Collapse" buttonAttrs={{ variant: 'link' }}>
```

Comment:
From looking at this line, is buttonAttrs the correct term to use for the prop? This looks more like props going into a react component as opposed to attributes you attach to the dom node. Does anyone else have an opinion?